src/pages/edit-test-center.page.js

```diff
@@ -211,154 +198,24 @@ export default class EditTestCenterPage extends Component {
-              </Article>
-              <Article>
-                <H3>Test Center Offerings</H3>
-                <div>
-                  <H4>Select all offerings that exist at this test center:</H4>
-                  <Checkboxes>
-                    {this.state.offerings.map((option, index) => (
-                      // TODO: Make checkbox partial
-                      <Checkbox
-                        key={option.key}
-                        className={option.value && 'checked'}
-                        onClick={this.onCheckboxSelected.bind(this, option)}
-                      >
-                        <img src={option.icon} alt={`Select ${option.displayName}`} style={{ margin: 'auto' }} />
-                        <label>{option.displayName}</label>
-                        <input readOnly hidden type="checkbox" id={option.key} checked={option.value}></input>
-                      </Checkbox>
-                    ))}
-                  </Checkboxes>
-                </div>
-                <div>
-                  <H4>Select all screening methods used at this test center:</H4>
-                  <Checkboxes>
-                    {this.state.screening.map((option, index) => (
-                      // TODO: Make checkbox partial
-                      <Checkbox
-                        key={option.key}
-                        className={option.value && 'checked'}
-                        onClick={this.onCheckboxSelected.bind(this, option)}
-                      >
-                        <img src={option.icon} alt={`Select ${option.displayName}`} style={{ margin: 'auto' }} />
-                        <label>{option.displayName}</label>
-                        <input readOnly hidden type="checkbox" id={option.key} checked={option.value}></input>
-                      </Checkbox>
-                    ))}
-                  </Checkboxes>
-                </div>
-              </Article>
-              <ButtonContainer>
-                <CancelButton onClick={this.onCancelClicked}>Cancel</CancelButton>
-                <SubmitButton color="primary" type="submit">
-                  Submit Test Center
-                </SubmitButton>
-              </ButtonContainer>
-            </form>
-          </Container>
+          <TestCenterUpdate 
+            title={'Edit Existing Test Center'} 
+            content={
+              `Edit the following form to update a test center within AllClear. All submissions will be reviewed
+              by the AllClear team within 24 hours.`
+            }
+            state={this.state} 
+            onCheckboxSelected={(option) => this.onCheckboxSelected.bind(this, option)} 
+            handleSnackbarClose={() => this.handleSnackbarClose} 
+            handleSubmit={(e) => this.handleSubmit(e)} 
+            onCancelClicked={() => this.onCancelClicked} 
+            onNameChange={(event) => this.setState({ postData: { ...this.state.postData, name: event.target.value } })} 
```

Comment:
you could generalize this into onChange and just pass the name of the item youre trying to change